contracts/bond/Bond.sol

```diff
@@ -150,34 +159,39 @@ contract Bond is
         }
 
         if (_hasBeenSlashed()) {
-            _redemptionRatio = _calculateRedemptionRation();
+            _redemptionRatio = _calculateRedemptionRatio();
         }
     }
 
     /**
+     * @notice Deposit swaps collateral tokens for an equal amount of debt tokens.
+     *
      * @dev Before the deposit can be made, this contract must have been approved to transfer the given amount
      * from the ERC20 token being used as collateral.
+     *
+     * @param amount_ The number of collateral token to transfer from the _msgSender().
+     *          Must be in the range of one to the number of debt tokens available for swapping.
+     *          The _msgSender() receives the debt tokens.
      */
-    function deposit(uint256 amount) external whenNotPaused whenNotRedeemable {
-        require(amount > 0, "Bond: too small");
-        require(amount <= _debtTokensRemaining(), "Bond: too large");
+    function deposit(uint256 amount_) external whenNotPaused whenNotRedeemable {
```

Comment:
Why make arguments strictly with "_"? To differentiate it from local variables? Is it a recommended codestyle?

Reply:
This is just a personal style and consistency within this contract.

Some function I'd used the underscore to avoid shadowing contract fields or inner function fields.

Really I'm just a fan of consistency ...if people feel strongly about using the suffix underscore, I'm always happy to follow a convention against using it, but otherwise 🤷

Reply:
Thinking about it, this actually might a code style from the TypeScript / JavaScript world, rather than Solidity 🤔 

Perhaps we shouldn't use the underscore suffix at all. Static analysers to check for variable shadowing, meaning a unique name is probably favoured.

Reply:
FYI, after reflecting on this and looking around online, I'm removing the suffix underscore.

Private variables keep the underscore prefix, but function parameters have no underscore.

The Slither static analysis flags variable showing (names too similar), with a single underscore not creating a big enough difference.